seacrowd/sea_datasets/bhinneka_korpus/bhinneka_korpus.py

```diff
@@ -0,0 +1,181 @@
+        schema = self.config.schema
+        features = datasets.Features(
+            {
+                "source_sentence": datasets.Value("string"),
+                "target_sentence": datasets.Value("string"),
+                "source_lang": datasets.Value("string"),
+                "target_lang": datasets.Value("string")
+            } if schema == "source" else schemas.text2text_features
+            if schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}" else None
+        )
+        if features is None:
+            raise ValueError("Invalid config schema")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        languages = []
+        bhinneka_source_data = []
+
+        lang = self.config.name.split("_")[2]
+        if lang in _LANGUAGES:
+            bhinneka_source_data.append(Path(dl_manager.download(_URLS + f"{LANGUAGES_MAP[lang]}/{lang}.xlsx")))
+            languages.append(lang)
+        else:
+            for i in _LANGUAGES:
+                bhinneka_source_data.append(Path(dl_manager.download(_URLS + f"{LANGUAGES_MAP[i]}/{i}.xlsx")))
+                languages.append(i)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": bhinneka_source_data,
+                    "split": "train",
+                    "languages": languages
+                }
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str, languages: List[str]) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        dfs = pd.read_excel(filepath[0], index_col=0, engine="openpyxl")
+        if len(filepath) > 2:
+            for path2file in filepath[1:]:
+                dfs = pd.merge(dfs, pd.read_excel(path2file, index_col=0, engine="openpyxl"))
+
+        source_sents = dfs.drop(columns=["ind"])
+        target_sents = dfs["ind"]
+
+        for idx, (source, target) in enumerate(zip(source_sents.values, target_sents.values)):
+            if self.config.schema == "source":
+                example = {
+                    "source_sentence": process_list(list(source)),
+                    "target_sentence": process_list(target),
+                    "source_lang": process_list(languages),
+                    "target_lang": "ind"
+                }
+            elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+                example = {
+                    "id": str(idx),
+                    "text_1": process_list(list(source)),
+                    "text_2": process_list(target),
+                    "text_1_name": process_list(languages),
```

Comment:
Please create one example per language instead of doing lists as supplying list here does not conform to the t2t schema